NewForce-Cohort-9/tabloidmvc-josh-jacobs-jingleheimer-schmidt
# Sf branch ticket 10

# Description

Added feature to edit categories. You can now successfully edit and save changes to categories. [Trello Ticket #10](https://trello.com/c/CFbTqHZI)


## Type of change


# Testing Instructions

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a 
4. git checkout sf-branch-ticket-10
5. If you don't see the most recent version of my files, git pull origin sf-branch-ticket-10
6. If you haven't created the Tabloid MVC database, please do so.
7. Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-bit-o-honeys/blob/main/SQL/01_TabloidMVC_Create_DB.sql)
8. Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)
9. Click the execute with debugger solid green button to launch the app
10. If not logged in, please go to Login on the nav bar and log in with this email: admin@example.com
11. Click on “Categories” in the nav bar, this will redirect you to the /categories page.
12. Choose the “Edit” option on “Close Magic”.
13. Enter “CLOSE MAGIC EDIT” for category name and click submit.
14. You will be redirected back to the categories index page and you should see that “Close Magic” has been changed to “CLOSE MAGIC EDIT” at the top of the list.
15. Click on “Categories” in the nav bar, this will redirect you to the /categories page.
16. Choose the “Edit” option on “Close Magic”.
17. Enter “new edit for close magic” for category name, but this time hit “Back To List”
18. Your changes should NOT be saved and you should be returned to the category index.



# Checklist:
